zerolog/README.md

```diff
@@ -89,20 +89,21 @@ import (
 )
 
 // RequestIDHeaderValue value for the request id header
-const RequestIDHeaderValue = "X-Request-Id"
+const RequestIDHeaderValue = "X-Request-ID"
```

Comment:
![image](https://user-images.githubusercontent.com/97824201/201325826-e4a6b5d7-b434-43fa-80d5-9625737adb19.png)
Change it here too, the log will not be printed out.

Reply:
Fixed